systems/overview/12-Glossary.md

```diff
@@ -107,6 +111,10 @@ Tools which help when working with Wikidata. The full list of Wikidata tools can
 
 A textual description of the functionality provided by the system that captures actor-system interaction. It specifies how a user interacts with a system and how the system responds to the user actions.
 
+## Value
```

Comment:
Per a discussion in mattermost we should not refer to this as a "value" only a "datavalue".
per https://www.mediawiki.org/wiki/Wikibase/DataModel#Values

OR we should remove this other definition of "value" from docs.

Reply:
> Per a discussion in mattermost we should not refer to this as a "value" only a "datavalue".
per https://www.mediawiki.org/wiki/Wikibase/DataModel#Values
> 
> OR we should remove this other definition of "value" from docs.

As far as I know "value" / "statement value" is what it's more commonly referred to. I haven't heard anyone use "datavalue" as a business term.

Reply:
Fixed by moving Value to Datavalue and changing all links to it. I still believe, we should fix the docs at https://www.mediawiki.org/wiki/Wikibase/DataModel#Values. I now refer the Glossary entry to https://www.wikidata.org/wiki/Help:Statements#Values, which says very different things about what _Values_ are.